scripts/page_browse.js

```diff
@@ -400,12 +381,23 @@ function pageBrowse(params, storageKey, replaceUrl, mentorMode = false) {
                         imageWidget = makeImageWidget(imageDiv, storageKey);
                         stretchDiv.append(imageDiv, textDiv);
                         let theSplitter = viewSplitter(stretchDiv, storageKey);
-                        if(mentorMode) {
-                            // make a text widget with splitter
-                            textWidget = makeTextWidget(textDiv, storageKey, true, theSplitter.mainSplit.reSize);
-                        } else {
-                            textWidget = makeTextWidget(textDiv, storageKey);
+                        let topTextDiv = $("<div>");//.append(textArea);
```

Comment:
remove the commented code if not needed to save space (I know, not much, but can't hurt).

Reply:
Thanks, done this.